src/main/fc/cli.c

```diff
@@ -3193,6 +3196,18 @@ static void cliStatus(char *cmdline)
         hardwareSensorStatusNames[getHwGPSStatus()]
     );
 
+#ifdef USE_ESC_SENSOR
+    uint8_t motorCount = getMotorCount();
```

Comment:
There is a check missing to only print this in the case that the user has configured the ESC sensors - no point printing it when there is no data to print.